src/main/resources/values/values/strings.xml

```diff
@@ -45,7 +45,7 @@
     <string name="location_manga">Manga</string>
 
     <!-- Categories Menu -->
-    <string name="categories_delete_confirm">Do you wish to delete the category %1$s?</string>
+    <string name="categories_delete_confirm">Really delete the %1$s category?</string>
```

Comment:
```suggestion
    <string name="categories_delete_confirm">Are you sure you want to delete the %1$s category?</string>
```
This sounds better

Reply:
It is a bit naggy, and very long. "Are you sure" has fallen out of fashion.

Reply:
```suggestion
    <string name="categories_delete_confirm">Delete the %1$s category?</string>
```